cmd/dagger/run.go

```diff
@@ -20,7 +20,7 @@ import (
 )
 
 var runCmd = &cobra.Command{
-	Use:     "run [flags] COMMAND",
+	Use:     "run [flags] [command]",
```

Comment:
Maybe add `...` since it can have multiple arguments:
```suggestion
	Use:     "run [flags] [command]...",
```